packages/mux-player/README.md

```diff
@@ -117,6 +117,70 @@ Enable the [Google Cast](https://developers.google.com/cast) button in the contr
 <script defer src="https://www.gstatic.com/cv/js/sender/v1/cast_sender.js?loadCastFramework=1"></script>
 ```
 
+### Hiding controls
+
+By default, Mux Player will show all the controls available:
```

Comment:
Might worth mentioning that depending on display size and stream type, not everything may be shown.

Reply:
good call, will add something in those lines